client/blocks/login/index.jsx

```diff
@@ -177,7 +177,7 @@ class Login extends Component {
 				locale: this.props.locale,
 				twoFactorAuthType: 'link',
 				oauth2ClientId: this.props.currentQuery?.client_id,
-				redirectTo: this.props.currentQuery?.redirect_to,
+				redirectTo: this.props.redirectTo,
```

Comment:
@escapemanuele Here I'm fixing what I believe is a little bug introduced by #83704. Better to use the redirect URL coming from the `getRedirectToOriginal` selector rather than reading it from the `redirect_to` query arg directly. Other similar calls in this module, like `handleTwoFactorRequested` or `sendEmailLogin` also use this value.